src/shared/calendar/day-of-week.tsx

```diff
@@ -0,0 +1,17 @@
+import { cn } from '../utils'
+
+/**일~토 요일 보여주는 컴포너트 */
+export default function DayOfWeek() {
+  const week = ['일', '월', '화', '수', '목', '금', '토']
```

Comment:
컴포넌트는 리렌더링이 되기 때문에 작성하신 상수들이 계속 생성된다는 문제가 있어요. 사실 이러한 문제가 엄청나게 큰 영향을 주는건 아니지만, 그래도 consts로 분리하는게 좋다고 생각합니다. 그리고 상수 네이밍(WEEK)으로 따라주면 될 것 같아요.

> [!NOTE]
> 어떤 사람들은 굳이 분리할 필요없다고 말하는 분도 있네요. 이유는 코드를 파악할 때 분리되어 있으면 보기 어렵다고 하네요. 재사용성이 필요해보인다면 분리하는게 좋고, 재사용하지 않는다면, 컴포넌트 바로 위에 작성하면 되겠네요

```suggestion
  const WEEK = ['일', '월', '화', '수', '목', '금', '토']
```

Reply:
상수 파일로 따로 분리하지 않는다 해도 컴포넌트 위에 선언해주는게 좋다고 생각합니다

Reply:
+ 추가) WEEK 보단 DAYS_OF_WEEK가 더 직관적이겠네요!